src/main/java/mat/client/shared/MessageDelegate.java

```diff
@@ -211,7 +211,7 @@ public class MessageDelegate {
     private static final String MEASURE_POPULATION_BASIS = "Please select the measure's Population basis prior to packaging.";
     private static final String MEASURE_DESCRIPTION_UNDERSCORE = "Measure name must not contain '_' (underscore).";
     public static final String MEASURE_TRANSFER_EMPTY_PACKAGE = "Unable to transfer selected measure. Issues were encountered with its package. Please re-package the measure before requesting a transfer to MADiE. If problem continues, please contact the help desk.";
-    public static final String MEASURE_TRANSFER_SUCCESS_MESSAGE = "Measure is being processed and transferred to MADiE.";
+    public static final String MEASURE_TRANSFER_SUCCESS_MESSAGE = "Measure is being processed and transferred to MADiE. You will receive an email with the transfer status.";
```

Comment:
There are many reasons to believe that our automated generated emails may end up in Spam folder, idk if we want to address it here. to let users check their spam folder just in case.

Reply:
that is a good point. I'll talk to Nicole.

Reply:
okay, we'll be adding the note in the user guide about it which hardly people read.